### DISCUSSION TITLE
Syntax highlighting for markdown code blocks?

### DISCUSSION THREAD

**Discussion by josephs07** [freeplane/freeplane#1283] [Help] [answered]
I am trying to migrate my logseq notes to freeplane. 
Is there any way to provide syntax highlighting on the notes while using markdown code blocks like in [obsidian](https://help.obsidian.md/Editing+and+formatting/Basic+formatting+syntax) and logseq?

**Comment by dpolivaev** [comment, marked as answer]
I am afraid it is not possible with the libraries that I know and use.

**Reply by josephs07** [reply]
Not sure if [RSyntaxTextArea](https://bobbylight.github.io/RSyntaxTextArea/) is useful?
